packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -80,7 +80,8 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
 
     this._warnDeprecations({
       rootProps: undefined,
-      allowTabKey: 'handleTabKey'
+      allowTabKey: 'handleTabKey',
+      elementType: 'elementType'
```

Comment:
I think this should be `elementType: 'as'`

Reply:
that makes sense the value of the key here should be what's replacing the key. just pushed this change.